src/com/raghav/javabasics/Counter.java

```diff
@@ -0,0 +1,55 @@
+package com.raghav.javabasics;
+// words, char, lines
+
+public class Counter {
+    public static void main(String[] args) {
+      String paragraph = "Change an Array\n" +
```

Comment:
What happens if your input string is empty or a null value?
String paragraph=“”; or
String paragraph=null;